### PR TITLE
Implement Feed component

### PR DESCRIPTION
**Pull request by yvesnrb**
This PR implements the Feed component and associated story.

### CODE REVIEW

**Review by ajmnz — CHANGES_REQUESTED** [review]
I think user-subscribed scheduled rooms should be on top, but how would you like to go about this? @benawad

**Comment by yvesnrb** [comment]
> I think user-subscribed scheduled rooms should be on top, but how would you like to go about this? @benawad

In that case, should I just add a 'subscribed' boolean to the props interface of Feeds to handle that logic and let whatever is referencing the Feed component handle it?

**Comment by ajmnz** [comment]
> > I think user-subscribed scheduled rooms should be on top, but how would you like to go about this? @benawad
> 
> In that case, should I just add a 'subscribed' boolean to the props interface of Feeds to handle that logic and let whatever is referencing the Feed component handle it?

Kind of, but maybe we don't want to show more than x scheduled rooms or only rooms that are scheduled for today.

**Comment by yvesnrb** [comment]
> > > I think user-subscribed scheduled rooms should be on top, but how would you like to go about this? @benawad
> > 
> > 
> > In that case, should I just add a 'subscribed' boolean to the props interface of Feeds to handle that logic and let whatever is referencing the Feed component handle it?
> 
> Kind of, but maybe we don't want to show more than x scheduled rooms or only rooms that are scheduled for today.

Got it. Just let me now how the feed should work @benawad and I'll make the necessary changes.

File: kibbeh/src/ui/Feed.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  rooms,
+  emptyPlaceholder,
+}) => {
+  const [liveRooms, setLiveRooms] = useState<RoomCardProps[]>([]);
+  const [scheduledRooms, setScheduledRooms] = useState<RoomCardProps[]>([]);
+
+  useEffect(() => {
+    setLiveRooms(rooms.filter((room) => isPast(room.scheduledFor)));
+    setScheduledRooms(rooms.filter((room) => !isPast(room.scheduledFor)));
+  }, [rooms]);
```
Comment (benawad) [inline kibbeh/src/ui/Feed.tsx]: don't use useEffect and store rooms in state. Take rooms as a prop. Also don't worry about scheduled rooms for now.

**Comment by yvesnrb** [comment]
I left the divider commented out for later and just mapped trough the rooms prop for now. Let me know if that is sufficient.

**Review by benawad — APPROVED** [review]

**Pull request merged**